misc/check.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# check for libcairo2
+if [ -f /usr/include/cairo/cairo.h ]; then
```

Comment:
My thought was `/usr/include` is pretty standard for installing headers. I can add an or check for `/usr/local/include/cairo/cairo.h` as well that might be more robust. We can add locations as they come up if needed. If there are not other objections I think we should merge and do an initial release of this patch since I know of several cases blocking on the the `sudo` problem.